grouper/repositories/interfaces.py

```diff
@@ -44,3 +64,64 @@ def permissions_for_user(self, user):
     def user_has_permission(self, user, permission):
         # type: (str, str) -> bool
         pass
+
+
+class ServiceAccountRepository(object):
```

Comment:
I'm not sure it's worth creating interfaces and both graph and non-graph versions of these repositories now when we don't need them yet. I was avoiding the interface layer for new repositories until I really needed it, since in the long run we may be getting rid of, or at least greatly reducing, the scope of the graph.